customers/services/profile.py

```diff
@@ -67,24 +39,80 @@ def get_config_template():
             PROFILE_API_URL: str,
         }
 
-    def get_all_profiles(self) -> Dict[UUID, HelsinkiProfileUser]:
-        def _exec_query(after=""):
-            response = self.query(self.ALL_PROFILES_QUERY % (100, after))
-            edges = response.get("profiles", {}).get("edges", [])
+    def get_all_profiles(
+        self, profile_ids: List[UUID] = None
+    ) -> Dict[UUID, HelsinkiProfileUser]:
+        query = """
+            query GetProfiles {{
+                profiles(serviceType: BERTH, first: {first}, after: "{after}", id: {ids}) {{
```

Comment:
As I understood they have not released those changes? If I understood correctly they're planning on moving that to production on the next months, but it's not there yet. And even when they do, it will still be accepted.

So I think for now it's safe to keep ti there until it's officially released, tested, and deprecated.

Reply:
OK!